bin/lint-migrations-file/src/lint_migrations_file.cljc

```diff
@@ -1,5 +1,6 @@
 (ns lint-migrations-file
   "This is cljc because it is used from both Clojure (:clj) and Babashka (:bb). Not cljs!"
+  #_:clj-kondo/ignore
```

Comment:
It's definitely better to ignore just one linter e.g. `#_{:clj-kondo/ignore [:discouraged-namespace]}` than to ignore everything. Also better to put the ignore before just the form you're ignoring rather than this entire thing. That way we can still catch other linter errors when people change this file in the future.